src/networking.c

```diff
@@ -1283,6 +1297,26 @@ void addReplySubcommandSyntaxError(client *c) {
     sdsfree(cmd);
 }
 
+/* Replies which were added after the client is blocked by a module are accumulated
+ * separately. We need to transmit those replies to the client. */
+void AddReplyModuleUnBlockedClient(client *c) {
```

Comment:
Nits:
1. The convention is to start with a lowercase letter. (AddReplyFromClient is the odd one.)
2. "unblocked" is one word, so it should have a lowercase "b".

```suggestion
void addReplyModuleUnblockedClient(client *c) {
```

Reply:
Done

Reply:
Done